.repo-metadata.json

```diff
@@ -1,13 +1,14 @@
 {
-  "name": "django-google-spanner",
-  "name_pretty": "Cloud Spanner Django",
-  "product_documentation": "https://cloud.google.com/spanner/docs/",
-  "client_documentation": "https://googleapis.dev/python/django-google-spanner/latest",
-  "issue_tracker": "https://issuetracker.google.com/issues?q=componentid:190851%2B%20status:open",
-  "release_level": "beta",
-  "language": "python",
-  "library_type": "INTEGRATION",
-  "repo": "googleapis/python-spanner-django",
-  "distribution_name": "django-google-spanner",
-  "requires_billing": true
+    "name": "django-google-spanner",
+    "name_pretty": "Cloud Spanner Django",
+    "product_documentation": "https://cloud.google.com/spanner/docs/",
+    "client_documentation": "https://googleapis.dev/python/django-google-spanner/latest",
+    "issue_tracker": "https://issuetracker.google.com/issues?q=componentid:190851%2B%20status:open",
+    "release_level": "preview",
```

Comment:
@parthea : Please change `release_level` to `ga`.
`"release_level": "ga"`

Reply:
Good catch! I've updated the release level to stable.